Author: Martin Pitt <mpitt@debian.org>
Description: Install lib files into /usr/lib/postgresql/<version>/lib/, so that several major versions can be installed in parallel.
Forwarded: No, Debian specific configuration with postgresql-common

Index: postgresql-9.0-9.0~beta1/src/Makefile.global.in
===================================================================
--- postgresql-9.0-9.0~beta1.orig/src/Makefile.global.in	2010-03-24 14:14:02.000000000 +0100
+++ postgresql-9.0-9.0~beta1/src/Makefile.global.in	2010-05-01 15:01:33.224729892 +0200
@@ -90,7 +90,7 @@
 pkglibdir = $(libdir)
 ifeq "$(findstring pgsql, $(pkglibdir))" ""
 ifeq "$(findstring postgres, $(pkglibdir))" ""
-override pkglibdir := $(pkglibdir)/postgresql
+override pkglibdir := $(pkglibdir)/postgresql/9.0/lib
 endif
 endif
 
